tests/chart_tests/test_platform_node_pools.py

```diff
@@ -38,6 +43,9 @@ class TestPodResources:
     )
     def test_pod_resources_configs(self, pod_spec):
         """Test that all pod spec have a nodeSelector,affinity and tolerations section defined."""
-        assert "nodeSelector" in pod_spec
-        assert "affinity" in pod_spec
-        assert "tolerations" in pod_spec
+        if pod_spec["containers"][0]["name"].split("release-name-")[-1] in ignore_list:
+            pytest.skip("Info: Resource doesn't adopt global nodepool" + pod_spec["containers"][0]["name"])
+        else:
+            assert "nodeSelector" in pod_spec
```

Comment:
Is it enough to assert that these exist? Shouldn't we ensure that they have some test content?

Reply:
added test cases for node-exporter in this PR  
cert-copy-and-toml-update  we have already covered the test cases
cert-copier we have not added support - should we added support